new PR template (#344)
TUM-Dev/NavigaTUM: .github/pull_request_template.md

Fixes #

## Proposed Changes (include Screenshots if possible)

-
-

## How to test this PR

1. Launch Subsystem XY
2. Click at
3. See changed XZ

## How has this been tested?

-
-

## Checklist:

> - [ ] This change requires a change in documentation
> - [ ] I have updated the documentation accordingly.

Reviewer: ```suggestion
- [ ] I have updated the documentation / No need to update the documentation
```
Can we merge them to one maybe? What would you say?
